### DISCUSSION TITLE
Unable to launch Wandb dashboard

### DISCUSSION THREAD

**Discussion by nayanjha16** [gokulkarthik/text2speech#5] [Q&A]
Hi @gokulkarthik , 

I am fairly new with Wandb , I was trying to implement this repo but I am getting the following error :

    self.dashboard_logger, self.c_logger = self.init_loggers( 
  File "/hdd4/Nayan/envs/Nayan_ai_or_bharat_tts/lib/python3.8/site-packages/trainer/trainer.py", line 575, in init_loggers
    dashboard_logger = logger_factory(config, output_path)
  File "/hdd4/Nayan/envs/Nayan_ai_or_bharat_tts/lib/python3.8/site-packages/trainer/logging/__init__.py", line 35, in logger_factory
    dashboard_logger = WandbLogger(  # pylint: disable=abstract-class-instantiated
TypeError: Can't instantiate abstract class WandbLogger with abstract methods add_audio, add_figure, add_scalar

Can you please guide me with how to resolve this issue

**Comment by gokulkarthik** [comment]
Hi, I guess this is because of the version mismatch of TTS package. Please follow the exact setup instructions in the README to avoid this issue. Specifically, ensure that you have followed the step 4 in [setup instructions in README](https://github.com/gokulkarthik/text2speech#setup )
